windows/en-US/firefox.adml

```diff
@@ -23,6 +23,10 @@
       <string id="Allow">Allowed Sites</string>
       <string id="Block">Blocked Sites</string>
       <string id="Authentication_SPNEGO">SPNEGO</string>
+      <string id="AppUpdateURL">URL for Firefox Updates</string>
+      <string id="AppUpdateURL_Explain">If this policy is enabled, you can set a URL to an other update server than the default. This could be helpful, if you run your own update server in your network and the clients have no direct access to the internet and the Mozilla update server.
```

Comment:
Sorry, one more tweak

If this policy is enabled, you can set a URL to an update server other than the default. This could be helpful if you run your own update server on your network.

Thanks!

Reply:
Sure, will fix.